siege_utilities/geo/spatial_transformations.py

```
@@ -314,12 +360,12 @@ def upload_spatial_data(self, gdf: GeoDataFrame, table_name: str, **kwargs) -> b
             # Upload data
             cursor = self.connection.cursor()
 
+            insert_stmt = _pg_sql.SQL(
+                "INSERT INTO {} (geom) VALUES (ST_GeomFromText(%s))"
+            ).format(qualified_ident)
             for idx, row in gdf.iterrows():
```

Ruff (0.15.12)

[warning] Loop control variable `idx` not used within loop body

Rename unused `idx` to `_idx`

(B007)